test/integration/expression-tests/heatmap-density/basic/test.json

```diff
@@ -1,5 +1,5 @@
 {
-  "expectExpressionType": {"kind": "Color"},
+  "propertySpec": {"type": "color", "default": "rgb(0,0,0)"},
```

Comment:
Might be more appropriate to have `default` here be a function (to exercise the special case handling in `getDefaultValue()`)